README.oracle_fdw

```diff
@@ -343,143 +352,143 @@ Oracle or PostgreSQL.
+The data type `geometry` is only available when PostGIS is installed.
 
-The only supported geometry types are POINT, LINE, POLYGON, MULTIPOINT,
-MULTILINE and MULTIPOLYGON in two and three dimensions.
+The only supported geometry types are `POINT`, `LINE`, `POLYGON`,
+`MULTIPOINT`, `MULTILINE` and `MULTIPOLYGON` in two and three dimensions.
 Empty PostGIS geometries are not supported because they have no equivalent
 in Oracle Spatial.
 
-NULL values for Oracle SRID will be converted to 0 and vice versa.
+`NULL` values for Oracle SRID will be converted to 0 and vice versa.
 For other conversions between Oracle SRID and PostGIS SRID, create a file
-"srid.map" in the PostgreSQL "share" directory.  Each line of this file
+`srid.map` in the PostgreSQL `share` directory.  Each line of this file
 shall contain an Oracle SRID and the corresponding PostGIS SRID, separated
 by whitespace.  Keep the file small for good performance.
 
 Support for IMPORT FOREIGN SCHEMA
 ---------------------------------
 
-From PostgreSQL 9.5 on, IMPORT FOREIGN SCHEMA is supported to bulk import
+From PostgreSQL 9.5 on, `IMPORT FOREIGN SCHEMA` is supported to bulk import
 table definitions for all tables in an Oracle schema.
-In addition to the documentation of IMPORT FOREIGN SCHEMA, consider the
+In addition to the documentation of `IMPORT FOREIGN SCHEMA`, consider the
 following:
 
-- IMPORT FOREIGN SCHEMA will create foreign tables for all objects found in
-  ALL_TAB_COLUMNS.  That includes tables, views and materialized views,
+- `IMPORT FOREIGN SCHEMA` will create foreign tables for all objects found in
+  `ALL_TAB_COLUMNS`.  That includes tables, views and materialized views,
   but not synonyms.
 
-- There are two supported options for IMPORT FOREIGN SCHEMA:
-  - "case": controls case folding for table and column names during import.
+- There are two supported options for `IMPORT FOREIGN SCHEMA`:
+  - `case`: controls case folding for table and column names during import.
     The possible values are:
-      "keep": leave the names as they are in Oracle, usually in upper case.
-      "lower": translate all table and column names to lower case.
-      "smart": only translate names that are all upper case in Oracle
+    - `keep`: leave the names as they are in Oracle, usually in upper case.
+    - `lower`: translate all table and column names to lower case.
+    - `smart`: only translate names that are all upper case in Oracle
                (this is the default).
-  - "readonly" (boolean): controls if imported tables can be modified.
-    If set to "true", all imported tables are created with the foreign
-    table option "readonly" set to "true" (see "Options" above).
-    The default is "false".
+  - `readonly` (boolean): controls if imported tables can be modified.
+    If set to `true`, all imported tables are created with the foreign
+    table option `readonly` set to `true` (see "Options" above).
+    The default is `false`.
 
```

Comment:
Bold like other options (but "true" and "false" should remain in double quotes).